docs/docs/shell/run-shell-commands.mdx

```diff
@@ -39,7 +39,7 @@ The `command` field is passed to `bash -c <command>`. The execution sandbox will
 
 The command is limited to operating on the specific set of input files provided as dependencies, and only produces output files for other targets to consume. It is not possible to mutate any file in the workspace.
 
-In case there are resulting files that should be captured and passed to any consuming targets, list them in the `outputs` field. To capture directories, simply add the path to the directory, with a trailing slash (as in the example `”files/”`, above).
```

Comment:
It would be a pleasure. Which part of it will get blasted away? This paragraph or is there something generating part of the docs coming from the other repo?

I'll give it a shot as soon as I can. Thanks!

Reply:
Just found it, please ignore my question.